lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -1352,6 +1365,14 @@ namespace Js
         numberConstructor = CreateBuiltinConstructor(&JavascriptNumber::EntryInfo::NewInstance,
             DeferredTypeHandler<InitializeNumberConstructor>::GetDefaultInstance());
         AddFunction(globalObject, PropertyIds::Number, numberConstructor);
+
+        if (scriptContext->GetConfig()->IsESBigIntEnabled())
+        {
+            bigIntConstructor = CreateBuiltinConstructor(&JavascriptBigInt::EntryInfo::NewInstance,
```

Comment:
Same question- should this be initialized in the case where bigint is disabled?

Reply:
will add that